media-gfx/openscad/openscad-2015.03_p3-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit elisp-common qmake-utils xdg-utils
 
```

Comment:
Please generate MY_PV from `${PV}` instead of hardcoding it.

Reply:
Changed.
This was a leftover from the older ebuild, I missed to address. Thanks for pointing it out.